var/spack/repos/builtin/packages/r-farver/package.py

```diff
@@ -13,4 +13,5 @@ class RFarver(RPackage):
     url      = "https://cloud.r-project.org/src/contrib/farver_2.0.1.tar.gz"
     list_url = "https://cloud.r-project.org/src/contrib/Archive/farver"
 
-    version('2.0.1', sha256='1642ca1519ef80616ab044ae7f6eaf464118356f2a7875e9d0e3df60ca84012b')
+    version('2.0.3', sha256='0e1590df79ec6078f10426411b96216b70568a4eaf3ffd84ca723add0ed8e5cc')
```

Comment:
@t-nojiri apologies but while we were discussing the hash change for 2.0.1, https://github.com/spack/spack/pull/20919 was merged which added 2.0.3 so there is now a conflict. To avoid the conflict, you can update the PR to just change 2.0.1.

Reply:
@scheibelp I merged the fixes.
Sorry for the late response.